RecoEgamma/PhotonIdentification/python/Identification/mvaPhotonID_Fall17_94X_V2_cff.py

```diff
@@ -0,0 +1,55 @@
+from RecoEgamma.PhotonIdentification.Identification.mvaPhotonID_tools import *
+#
```

Comment:
I think that the import of  `os.path` and the definition of `weightFileBaseDir` are missing here

Reply:
same with `category_cuts` later in the same file

Reply:
Oh yes, that's absolutely right! And I noticed I messed up with this PR (I forgot https://github.com/cms-sw/cmssw/pull/24131 was merged in 10_3_X), so I have to adapt the config files a bit.

Reply:
...actually we are talking about the backport https://github.com/cms-sw/cmssw/pull/25313. Nothing is missing here.

Reply:
@peruzzim for what I can see, they are all included within mvaPhotonID_tools, and indeed the unit tests using this fragment looks to work properly